### PR TITLE
[P2P] Remove NetworkAddress from the solution Part #1

### PR DESCRIPTION
**Pull request by fassadlr**
This is part 1 to remove Network Address from the solution.

### CODE REVIEW

**Comment by bokobza** [comment]
`endPoint` should be spelled `endpoint`.

File: src/Stratis.Bitcoin/P2P/PeerAddress.cs [hunk 1]
```diff
@@ -182,25 +181,25 @@ internal void SetLastSeen(DateTime lastSeenAt)
...
+                EndPoint = endPoint,
                 loopback = IPAddress.Loopback.ToString()
             };
         }
 
         /// <summary>
         /// Creates a new peer address instance and sets the loopback address (source).
         /// </summary>
-        /// <param name="address">The network address of the peer.</param>
+        /// <param name="endPoint">The end point of the peer.</param>
```
Comment (bokobza) [inline src/Stratis.Bitcoin/P2P/PeerAddress.cs]: `endpoint` in one word is more common.

**Comment by dangershony** [comment]
Generally this looks good all integration tests passed?

**Comment by bokobza** [comment]
smh
even here they write it `endpoint`
image

**Comment by bokobza** [comment]
Even IP should really be Ip (same for Api and Rpc).
Let's change the things on which we have control.

**Comment by fassadlr** [comment]
@bokobza all good dude! I have renamed it to `endpoint ` now 👍 

@dangershony Yes tests are passing 💯

**Comment by bokobza** [comment]
You're the best. 💪

**Pull request merged**